### PR TITLE
New jupyter-chaste version

### PR DESCRIPTION
**Pull request by elisabettai**
This PR ports the old jupyter-chaste to the dysidecar version. It also makes the code of the service open-source

- related to: https://github.com/ITISFoundation/osparc-issues/issues/694 
- related to: https://github.com/ITISFoundation/osparc-issues/issues/942

## How to test
Some instructions for manual testing have been added to the README.md

### CODE REVIEW

**Review by pcrespov — COMMENTED** [review]
:tada: 

pls. make sure to fill the PR labels

**Comment by elisabettai** [comment]
> tada
> 
> pls. make sure to fill the PR labels

Thanks @pcrespov. I am trying to fill in the PR labels, but I didn't find many I can fill in. I've added `Project`, but I can't assign a Milestone, for instance. Maybe that's only available for issues?

**Comment by pcrespov** [comment]
> > tada
> > pls. make sure to fill the PR labels
> 
> Thanks @pcrespov. I am trying to fill in the PR labels, but I didn't find many I can fill in. I've added `Project`, but I can't assign a Milestone, for instance. Maybe that's only available for issues?

@elisabettai  we;ll, you need to create the milestone in this repo. Since the repo is new, there are no milestones

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1,15 @@
+  
+# Maps code in repository with maintainers
+# Order is important. The last matching pattern has the most precedence.
+# SEE https://docs.github.com/en/free-pro-team@latest/github/creating-cloning-and-archiving-repositories/about-code-owners#example-of-a-codeowners-file
+
+
+# files and folders recursively
+Dockerfile          @elisabettai
+Makefile            @GitHK
```
Comment (pcrespov) [inline .github/CODEOWNERS]: check this!

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+<<<<<<< HEAD
+## [2.0.0] - 2023-06-21
+- updated to run via dynamic-sidecar
+- Base image now uses the same as jupyterlab-math
+
+=======
```
Comment (GitHK) [inline CHANGELOG.md]: looks like a bad merge here

File: CHANGELOG.md [hunk 3]
```diff
@@ -35,4 +41,5 @@ All notable changes to this service will be documented in this file. The format
   - Security in case of vulnerabilities.
 
 SEE https://keepachangelog.com/en/1.0.0/
--->
\ No newline at end of file
+-->
+>>>>>>> 369a70b1b8723f5911e2c589528a0d8182652bdc
```
Comment (GitHK) [inline CHANGELOG.md]: also here

**Review by GitHK — COMMENTED** [review]
some things I've noticed

File: .github/CODEOWNERS (hunk 1 above)
Comment (elisabettai) [reply .github/CODEOWNERS]: I've cleaned it up. I've also removed the Dockerfile in the parent directory (it was a left over from the previous version we had on GitLab).

File: CHANGELOG.md (hunk 3 above)
Comment (elisabettai) [reply CHANGELOG.md]: Thanks. I had indeed forgot to save the file after solving the merge conflict. 🙈 
Should be fixed now.

File: .github/ISSUE_TEMPLATE/ask_question.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
+---
```
Comment (pcrespov) [inline .github/ISSUE_TEMPLATE/ask_question.md:1]: TIP: I would convert all these `ISSUE_TEMPLATE`s to issue forms  and add them in the cookiecutter. It simplifies interaction with users and in the near future we will add support links to the metadata.

https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository#creating-issue-forms

**Review by pcrespov — APPROVED** [review]

**Pull request merged**